website/docs/artist_hosts_maya.md

```diff
@@ -322,7 +322,8 @@ Example setup:
 - **Handle Start**: additional frames to export at frame end. Ei. frame end + handle end = export end.
 - **Step**: frequency of sampling the export. For example when dealing with quick movements for motion blur, a step size of less than 1 might be better.
 - **Refresh**: refresh the viewport when exporting the pointcache. For performance is best to leave off, but certain situations can require to refresh the viewport, for example using the Bullet plugin.
-- **Attr**: specific attributes to publish separated by `;`
+- **Attr**: specific attributes to publish separated by `;`.
+- **AttrPrefix**: Prefix filter for determining which geometric attributes to write out, separated by `;`.
```

Comment:
Sorry to be precise, but it's not only geometric - it'd be any attribute that matches it, also on transforms.

```suggestion
- **Attr**: specific attributes to publish separated by `;`.
- **AttrPrefix**: specific attributes which start with this prefix to publish separated by `;`.
```

Or:
```suggestion
- **Attr**: include attributes with these names in the publish separated by `;`, for example `my_attribute;other_attribute`
- **AttrPrefix**: include attributes starting with this prefix in the publish separated by `;`, for example `ai;vray`
```